noxfile.py

```diff
@@ -75,7 +75,10 @@ def default(session):
     session.install(
         "mock", "pytest", "pytest-cov",
     )
-    session.install("-e", ".[fastavro,pandas,pyarrow]")
+    extras = "fastavro,pandas,pyarrow"
```

Comment:
I know `synthtool` will just revert this change, but I figure we can manually keep this in until pyarrow 3.0 is out.